app/code/community/Drip/Connect/Model/Transformer/Order.php

```diff
@@ -188,6 +192,10 @@ protected function getOrderShippingData()
      */
     protected function getOrderAddressData($addressId)
     {
+        if ($addressId === null) {
+            return;
+        }
```

Comment:
I'm not sure that this fix exists in the M2 version of the PR. Do we think it applies there?

Reply:
It looks like the fix there is slightly different. This is checking for a `null` return from `getShippingAddressId`. The M2 fix checks the products for the type `virtual`.

Not sure if this would work over there...

Reply:
The two have already diverted in the way they retrieve and handle shipping address data, these PRs actually bring M1 and M2 closer together in behavior, even if the implementation is slightly different.